.eslintrc.cjs

```diff
@@ -5,8 +5,7 @@ module.exports = {
     es2021: true,
     node: true,
   },
-  // TODO: we need to move to the following, but this gets us started with infrastructure without breaking our build
-  // extends: [ "eslint:recommended", "plugin:@typescript-eslint/eslint-recommended"],
+  extends: [ "eslint:recommended", "plugin:@typescript-eslint/eslint-recommended"],
   extends: ["plugin:@typescript-eslint/eslint-recommended"],
```

Comment:
Just realized that the lower version totally overwrites the upper version.  That explains why I didn't pick up any issues from it.

Removing the bottom line and leaving the pair of 'extends' settings _definitely_ turns up some issues, some of them rooted in the differences between JS and TS.